rust_dev_preview/sitewise/README.md

```diff
@@ -0,0 +1,60 @@
+# AWS SDK for Rust code examples for IoT SiteWise
+
+## Purpose
+
+These examples demonstrate how to perform several AWS IoT SiteWise operations using the developer preview version of the AWS SDK for Rust.
+
+AWS IoT SiteWise is a managed service that simplifies collecting, organizing, and analyzing industrial equipment data.
+
+## Code examples
+
+- [List assets](src/bin/list-assets.rs) (ListAssets)
+
+## ⚠ Important
+
+- We recommend that you grant this code least privilege, 
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
+
+## Running the code examples
+
+### Prerequisites
+
+You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+
+### list-assets.rs
+
+This example lists IoT SiteWise assets in the Region.
+
+`cargo run --bin list-assets -- -f FILTER [-a ASSET-MODEL-ID] [-r REGION] [-v]`
+
+- _FILTER_ is the type of filter. It must be one of:
```

Comment:
- _FILTER_ is the type of filter. It must be one of the following:

Reply:
updated